### PR TITLE
UI changes, basic functionality

### PR DESCRIPTION
**Pull request by pedro-silva-oliveira**
this adds some basic UI and functionality. 

It's currently still missing:
- Counter resets after battle. either auto ou manual?
- Sorting the dictionary from highest value to lowest

### CODE REVIEW

File: GreedyCounter/Form1.cs [hunk 1]
```diff
@@ -111,5 +160,29 @@ private void AppendTextToOutput(string text)
...
+        {
+
+            //updates the Output text box
+            sb.Clear();
+            sb.AppendLine("Greedy Count:");
+            // Add the keypair to a string via stringbuilder
+            foreach (KeyValuePair<string, int> kvp in cDict)
+            {
+                //sb.AppendFormat("{0}: {1} \n", kvp.Key, kvp.Value.ToString());
+                sb.AppendLine(kvp.Key + ": " + kvp.Value.ToString());
```
Comment (DanielBunting) [inline GreedyCounter/Form1.cs]: [nit] I would use string interpolation here. This line could be ```sb.AppendLine($"{kvp.Key} : {kvp.Value}");``` Which I think looks nicer.

File: GreedyCounter/Form1.cs [hunk 2]
```diff
@@ -87,17 +115,38 @@ private string UserPerformedStrike(string input)
             if ((input.Contains("delivers an overwhelming barrage against") && input.Contains(", causing some treasure to fall from their grip!"))
                 || (input.Contains("swings a devious blow against") && input.Contains(", jarring some treasure loose!"))
                 || (input.Contains("executes a masterful strike against ") && input.Contains(", who drops some treasure in surprise!"))
-                || (input.Contains("performs a powerful attack against ") && input.Contains(", and steals some loot in the process!")))
+                || (input.Contains("performs a powerful attack against ") && input.Contains(", and steals some loot in the process!"))
+                || (input.Contains("smiles"))) //this last condition is to be removed. Serves for testing only
```
Comment (DanielBunting) [inline GreedyCounter/Form1.cs]: [nit] Would be better to remove in the pull request here.

File: GreedyCounter/Form1.cs [hunk 3]
```diff
@@ -10,11 +11,29 @@ public partial class Form1 : Form
...
+        StringBuilder sb = new StringBuilder();
+
         public Form1()
         {
             InitializeComponent();
+
+            timer = new System.Windows.Forms.Timer();
+            timer.Interval = 1000;
+            timer.Tick += new EventHandler(timer_Tick);
+            timer.Start();
```
Comment (DanielBunting) [inline GreedyCounter/Form1.cs]: [Query] Should we start the timer before we've selected a file? Could there be potential issues with this?

File: GreedyCounter/Form1.cs (hunk 1 above)
Comment (pedro-silva-oliveira) [reply GreedyCounter/Form1.cs]: Oh nice! I agree

File: GreedyCounter/Form1.cs (hunk 2 above)
Comment (pedro-silva-oliveira) [reply GreedyCounter/Form1.cs]: How do I do this in order for the line not to part of the pull request?

File: GreedyCounter/Form1.cs (hunk 3 above)
Comment (pedro-silva-oliveira) [reply GreedyCounter/Form1.cs]: Honestly, I have no idea. But starting the timer to after the log file has been selected seems ok too.

**Review by DanielBunting — APPROVED** [review]

File: GreedyCounter/Form1.cs (hunk 2 above)
Comment (DanielBunting) [reply GreedyCounter/Form1.cs]: Just delete the line on your local fork, push the code, and it'll remove it here!

Comment (pedro-silva-oliveira) [reply GreedyCounter/Form1.cs]: Hum.. I just commented it out. Good enough?

Comment (DanielBunting) [reply GreedyCounter/Form1.cs]: Perfect!

**Pull request merged**